.coafile

```diff
@@ -66,3 +66,7 @@ cs_keywords =
 [PythonPackageInit]
 bears = PythonPackageInitBear
 ignore = *.py
+
+[yml]
+bears = YAMLLintBear
+files = *.yml, .ci/*.yml, tests/**/*.yml, bears/*.yml
```

Comment:
Okay, we don't have any `.yml` files inside the bears directory since all bears are written in Python so we can remove `bears/*.yml`. However, we do have `.yml` files inside the `tests` directory so we need to keep `tests/**/*.yml`.